src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResultItem.tsx

```diff
@@ -128,15 +134,19 @@ const LargeAuctionItem: SFC<Props> = props => {
           justifyContent="center"
           height="80px"
           width="80px"
-          pr={2}
         >
-          <StyledImage src={imageUrl} preventRightClick />
+          {imageUrl && (
+            <StyledImage
+              src={imageUrl}
+              Fallback={() => renderFallbackImage()}
+            />
+          )}
```

Comment:
I was playing with storybooks and I noticed some empty images. 

<img width="373" alt="image" src="https://user-images.githubusercontent.com/3087225/77177990-d2111a00-6a9c-11ea-8546-2e1f530fe659.png">

I think the `imageUrl &&` part is the problem. 

I think what you want to do here is something like....

```
imageUrl ? <StyledImage ... /> : renderFallbackImage()
```